src/clr-angular/data/datagrid/datagrid-action-overflow.ts

```diff
@@ -50,6 +75,18 @@ export class ClrDatagridActionOverflow implements OnDestroy {
     if (boolOpen !== this._open) {
       this._open = boolOpen;
       this.openChanged.emit(boolOpen);
+      if (boolOpen && isPlatformBrowser(this.platformId)) {
+        this.zone.runOutsideAngular(() => {
+          setTimeout(() => {
+            const firstButton = this.elementRef.nativeElement.querySelector(
+              `.${this.overflowClassName} button.action-item`
```

Comment:
Now we are targeting specific buttons in the component so we don't `.${this.overflowClassName}` part and have the class on the wrapper div element directly.

Reply:
Yeah, it will be easier to read and maintain. I'll add it to the next PR, that will handle the "select all" issue.